Application/Features/BookingFeature/BookingProfile.cs

```diff
@@ -12,12 +12,27 @@ public BookingProfile()
             CreateMap<Booking, BookingDto>()
                 .ForMember(dest => dest.UserName, opt => opt.MapFrom(src => src.User.UserName));
 
+            // Is it ok to return all information thi way? or should we query the database separately for each required information?
+            // And why do we have a full-info response when we create a modeinstead of returning just an id? So he can then call get-by-id-query.
```

Comment:
anyway you will have to call getById to grab info and do not copy-paste it to create|update